src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -36,7 +36,7 @@ namespace System.Buffers
         /// <summary>
         /// Determines if the <see cref="ReadOnlySequence{T}"/> is empty.
         /// </summary>
-        public bool IsEmpty => Length == 0;
+        public bool IsEmpty => IsEmpty;
```

Comment:
Theoretically tests should find all of these infinite recursions, but I feel like it's still virtuous to point them out.